js/bit4you.js

```diff
@@ -0,0 +1,624 @@
+        } else {
+            request.address = address;
+        }
+        request.chain = params.network + '-mainnet' || '';
+        try {
+            const response = await this.privatePostWalletSend (this.extend (request));
+            return response || [];
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async fetchStatus (params = {}) {
+        return {
+            'status': 'ok', // 'ok', 'shutdown', 'error', 'maintenance'
+            'updated': this.milliseconds (), // integer, last updated timestamp in milliseconds if updated via the API
+            'eta': undefined, // when the maintenance or outage is expected to end
+            'url': undefined, // a link to a GitHub issue or to an exchange post on the subject
+        };
+    }
+
+    async fetchTicker (symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        const cached = this.market (symbol);
+        const market = cached['symbol'];
+        try {
+            const response = await this.fetchTickers ();
+            return response[market] || [];
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        try {
+            const response = await this.publicGetMarketSummaries ();
+            const result = {};
+            const tickerIds = Object.keys (response);
+            for (let i = 0; i < tickerIds.length; i++) {
+                const key = tickerIds[i];
+                const ticker = response[key];
+                const symbol = this.safeString (ticker, 'market').replace ('-', '/');
+                result[symbol] = {
+                    'symbol': symbol,
+                    'info': ticker,
+                    'timestamp': undefined,
+                    'datetime': undefined,
+                    'high': this.safeFloat (ticker, 'high'),
+                    'low': this.safeFloat (ticker, 'low'),
+                    'bid': this.safeFloat (ticker, 'bid'),
+                    'bidVolume': undefined,
+                    'ask': this.safeFloat (ticker, 'ask'),
+                    'askVolume': undefined,
+                    'vwap': undefined,
+                    'open': this.safeFloat (ticker, 'open'),
+                    'close': this.safeFloat (ticker, 'last'),
+                    'last': this.safeFloat (ticker, 'last'),
+                    'previousClose': this.safeFloat (ticker, 'prevDay'),
+                    'change': undefined,
+                    'percentage': undefined,
+                    'average': undefined,
+                    'baseVolume': this.safeFloat (ticker, 'volume'),
+                    'quoteVolume': undefined,
+                };
+            }
+            return result;
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
`this.extend()` unifies two dicts so you should be doing `this.extend (request, params`) instead